libs/output-mapping/src/Storage/NativeTypeDecisionHelper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Storage;
+
+use Keboola\StorageApiBranch\StorageApiToken;
+
+class NativeTypeDecisionHelper
+{
+    public static function shouldEnforceBaseTypes(StorageApiToken $token, string $backend): bool
+    {
+        return !$token->hasFeature('bigquery-native-types') && $backend === 'bigquery';
```

Comment:
featury máme všechny tady - https://github.com/keboola/platform-libraries/blob/main/libs/output-mapping/src/OutputMappingSettings.php
měl by si to volat stejně

Reply:
Tak jsem to původně měl, ale pak jsem to vyhodil, protože bych musel probublat OutputMappingSettings. Teď ale koukám, že výsledný PR vypadá trochu jinak než jsem to měl původně a asi by to šlo :) Zkusím to.